Extensions/AuthPropertiesTokenCacheHelper.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Identity.Client;
+
+namespace TodoListService.Extensions
+{
+    public class AuthPropertiesTokenCacheHelper
+    {
+        private const string TokenCacheKey = ".UserTokenCache";
+
+        private HttpContext _httpContext;
+        private ClaimsPrincipal _principal;
+        private AuthenticationProperties _authProperties;
+        private string _signInScheme;
+        public TokenCache TokenCache { get; private set; }
```

Comment:
Constructors should be before Properties.

Reply:
Made this change.